ui/views/component/dialog-view.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
```

Comment:
We should use a shared dialog component for this purpose. Let's make a standard UI component for it.